docker/Dockerfile

```diff
@@ -1,11 +1,33 @@
+ARG ALPINE_VERSION=3.19
+ARG NGINX_VERSION=1.27.0
+ARG PYTHON_VERSION=3.11
+
 # Build frontend
-FROM node:lts-alpine as front-build-stage
+FROM node:lts-alpine${ALPINE_VERSION} as front-build-stage
 WORKDIR /front
 COPY ./frontend ./
 RUN npm install && npm run build
 
+# Build backend environment
+FROM python:${PYTHON_VERSION}-alpine${ALPINE_VERSION} AS backend-build
+
+RUN apk add --no-cache \
+        gcc \
+        mariadb-connector-c-dev \
+        musl-dev
+
+RUN pip install poetry
```

Comment:
We use poetry in non-package mode, can i be installed like this now that 1.8.0 is live, or do we still need to use `pip install --no-cache git+https://github.com/radoering/poetry.git@non-package-mode`?

https://python-poetry.org/blog/announcing-poetry-1.8.0/#better-support-for-non-package-projects

Reply:
Exactly, `poetry` 1.8.0 now supports `package-mode = false`, so we can move to upstream and stop relying on that fork.